Ical.Net/Evaluation/Evaluator.cs

```diff
@@ -69,11 +60,5 @@ protected void IncrementDate(ref DateTime dt, RecurrencePattern pattern, int int
 
     public System.Globalization.Calendar Calendar { get; private set; }
 
-    public virtual ICalendarObject AssociatedObject
-    {
-        get => _mAssociatedObject;
-        protected set => _mAssociatedObject = value;
-    }
-
-    public abstract IEnumerable<Period> Evaluate(IDateTime referenceDate, DateTime? periodStart, DateTime? periodEnd, bool includeReferenceDateInResults);
+    public abstract IEnumerable<Period> Evaluate(IDateTime referenceDate, IDateTime? periodStart, IDateTime? periodEnd, bool includeReferenceDateInResults);
```

Comment:
nullable arguments should have `#nullable enable`. The effort shouldn't be to big?

Reply:
Oh yes. It actually wasn't my intention to introduce nullable, but now that it's already there, I tried to enable `#nullable` for the affected files :-D I'm not using it too much, so please have a close look!